### ISSUE TITLE
Scroll issue in shifting in facilities.

### ISSUE THREAD

**Issue by dwip708** [coronasafe/care_fe#7487]
**Issue**
In shifting in facilities, the scroll is common for all the fields like destination approved, transportation arranged, transferin progress , etc. This leads to an issue that if one field has more patients than others, the scroll for others goes below the max no if patients in that. If any one has 100 patients, others have 2 each, after viewing 100th patient of one, you need to scroll back up for the entire to view the ones for other fields. This is a UI issue and needs to be addressed.
@rithviknishad @nihal467 

![issue](https://github.com/coronasafe/care_fe/assets/97839370/eee49fec-861d-4877-82c8-a5bc652e354f)

**Solution**
Separate scroll for each field like destination approved, transportation arranged, transfer-in progress , etc.

**Comment by kumar11jr** [comment]
hey @rithviknishad  can you assigned me this issue.

**Comment by dwip708** [comment]
Hey @rithviknishad ,
I found a bug in the shifting section. I had done the scroll thing but the issue is that even if there are more than 14 patients in any section, only 14 are loaded by pagination. THERE IS NO MORE OPTION. It is expected to have a more option at the end, but there is some bug in the code so it is not showing, EVEN IN ORIGINAL HOSTED WEBSITE. I am working on that, I will sort it real soon.  I have shared a ss of the bug. 

![image](https://github.com/coronasafe/care_fe/assets/97839370/4692662b-5bae-40f6-846c-cece306bd434)
